moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame.cpp

```diff
@@ -445,6 +445,26 @@ void MotionPlanningFrame::importResource(const std::string& path)
         return;
       }
 
+      // Center the mesh so scaling applies correctly
```

Comment:
Centering is not required if no scaling will be applied, is it?
You can merge this code with the scaling code instead.